src/components/App/App.js

```diff
@@ -98,55 +134,90 @@ const App = () => {
   /******************************************************************************************** */
   /** ***************************** Handles `Register` & `Login` Logic *************************** */
 
-  const onRegister = ({ email, password, username }) => {
+  const onRegister = ({ email, password, name }) => {
     auth
-      .register(email, password, username)
+      .register(email, password, name)
       .then((res) => {
-        setIsSuccessOpen(true);
+        if (res.status === 201) {
+          setIsRegisterOpen(false);
+          setIsSuccessOpen(true);
+        }
+
+        if (res.status === 409 || 500 || 400) {
```

Comment:
`        if (res.status === 409 || 500 || 400) {
`

this check doesn't work properly


you can't check `409 || 500 || 400` because they are alwasy `true`


either you need to check them separetly 

```js

        if (res.status === 409 || res.status === 500 || res.status === 400) {


```


or you don't need them at all. You'll check each response with `res.ok` and that's enough (in your api files)


**please, correct it everywhere**